src/index.js

```diff
@@ -4,6 +4,22 @@ import bindActionCreators from './utils/bindActionCreators'
 import applyMiddleware from './utils/applyMiddleware'
 import compose from './utils/compose'
 
+/*
+* create a function so that we can check if the function name has been altered by minification
+* if the function has been minified and NODE_ENV !== 'production', warn the user
+*/
+function isCrushed() {}
+
+if (isCrushed.name !== 'isCrushed' && process.env.NODE_ENV !== 'production') {
```

Comment:
This doesn't seem right, if `process.env.NODE_ENV` is not defined that doesn't necessarily indicate that I'm using minified code outside of production, or am I missing something? In our codebase we define `NODE_ENV` without the "process.env" prefix but I can imagine lots of apps have no need to define it.